sound/soc/sof/pm.c

```diff
@@ -128,7 +128,8 @@ static int sof_resume(struct device *dev, bool runtime_resume)
 	return ret;
 }
 
-static int sof_suspend(struct device *dev, bool runtime_suspend)
+static int sof_suspend(struct device *dev, bool runtime_suspend,
+		       enum sof_dsp_power_state dsp_power_target)
```

Comment:
Oh my. so now we are dealing with D0i3 in two places, in snd_sof_dsp_suspend and sof_suspend.

Edit: but wait, this is changed later. REALLY HARD TO REVIEW!